misc/scripts/install-local.sh

```diff
@@ -28,6 +28,8 @@ function cleanup () {
 	if [ -f /tmp/pacstall-optdepends ]; then
 		sudo rm /tmp/pacstall-optdepends
 	fi
+
+	rm -rf "/tmp/pacstall/pacscripts/"
```

Comment:
We already nuke `/tmp/pacstall`, there is no need to nuke it again. It'll just print an error

Reply:
alright, on it.